state/state.go

```diff
@@ -48,7 +48,13 @@ type DynamoDBStater struct {
 	BackfillInterval time.Duration
 }
 
-func NewDynamoDBStater(session *session.Session, service string, backfillHrs int) *DynamoDBStater {
+func NewDynamoDBStater(session *session.Session, service string, backfillHrs int) (*DynamoDBStater, error) {
+	stater := DynamoDBStater{
```

Comment:
In most Go code I've seen you'd have type `&DynamoDBStater` here (i.e., declaring your intention for this to be a pointer up-front, vs. getting the address when you need to) and return `stater` directly. Can you please update to have it this way?